src/coord/src/session/vars.rs

```diff
@@ -476,16 +476,18 @@ impl Vars {
             server_version_num: _,
             sql_safe_updates,
             standard_conforming_strings: _,
-            timezone: _,
+            timezone,
             transaction_isolation: _,
         } = self;
         application_name.end_transaction(action);
         client_min_messages.end_transaction(action);
+        cluster.end_transaction(action);
         database.end_transaction(action);
+        extra_float_digits.end_transaction(action);
         qgm_optimizations.end_transaction(action);
         search_path.end_transaction(action);
-        extra_float_digits.end_transaction(action);
         sql_safe_updates.end_transaction(action);
+        timezone.end_transaction(action);
     }
```

Comment:
Oy, this is quite a footgun.

Reply:
I was wondering if there is something we could do to make this less likely in the future, but probably not without a significant refactor.